aws-mssql.yml

```diff
@@ -120,6 +120,17 @@ provision:
       MSSQL offers additional features such as SQL Server Audit, Transparent Data Encryption, etc.
       See AWS RDS options for the Microsoft SQL Server database engine for details:
       https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.SQLServer.Options.html
+  - field_name: parameter_group_name
+    type: string
+    default: ""
+    details: |
+      Name of the custom parameter group to associate with this instance.
+      If left unset, the default parameter group for the specified mssql_version is used.
```

Comment:
nit-pick: the parameter group used is what we created with the containment database configuration